### PR TITLE
feat: implement minmax files count limitation

### PR DESCRIPTION
**Pull request by nd0ut**

### CODE REVIEW

File: blocks/themes/uc-basic/l10n.css [hunk 1]
```diff
@@ -78,4 +78,8 @@
   --l10n-camera-permissions-denied: 'Camera access denied';
   --l10n-camera-permissions-prompt: 'Camera access prompted';
   --l10n-camera-permissions-request: 'Request permissions';
+
+  --l10n-files-count-limit-error-title: 'Files count limit overflow';
+  --l10n-files-count-limit-error-too-few: 'You’ve chosen {{total}}. At least {{min}} required.';
+  --l10n-files-count-limit-error-too-many: 'You’ve chosen too many files. {{max}} is maximum.';
```
Comment (foxeyes) [inline blocks/themes/uc-basic/l10n.css:84]: "Not enough files, select more..." & "Too many files selected..."? Maybe.

File: utils/stringTemplate.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      let value = hasValue ? variables[key].toString() : KEY_NOT_FOUND;
+      !hasValue && missingKeys.push(key);
+
+      result += value;
+      result += text;
+    }
+  }
+
+  return { string: result, missingKeys: missingKeys };
+}
```
Comment (foxeyes) [inline utils/stringTemplate.js:29]: Looks overcomplicated a bit )
```javascript
/**
 * 
 * @param {String} template
 * @param {Object<string, string>} data 
 * @param {String} [openToken]
 * @param {String} [closeToken]
 */
export function applyTemplateData(template, data, openToken = '{{', closeToken = '}}') {
  for (let key in data) {
    template = template.replaceAll(openToken + key + closeToken, data[key]);
  }
  return template;
}
```

**Review by foxeyes — APPROVED** [review]

File: utils/stringTemplate.js (hunk 2 above)
Comment (nd0ut) [reply utils/stringTemplate.js]: yeah, you're right

File: blocks/themes/uc-basic/l10n.css (hunk 1 above)
Comment (nd0ut) [reply blocks/themes/uc-basic/l10n.css:84]: I think end-user should know the limits

Comment (foxeyes) [reply blocks/themes/uc-basic/l10n.css:84]: Totally agree, but what i mean is the other part of the message. Just a phrase construction...

**Pull request merged**